xcell/mappers/mapper_DELS.py

```diff
@@ -56,6 +56,7 @@ def _get_catalog(self):
                 cat.append(Table.read(f))
         cat = vstack(cat)
         cat = self._bin_z(cat)
+        cat.keep_columns(['RA', 'DEC', self.pz])
```

Comment:
Wouldn't move this line before the `vstack` make the operation faster since you will operate over less columns?

Reply:
Done